redirects-transition.conf

```diff
@@ -1296,19 +1296,25 @@ rewrite ^/press/foia.shtml https://www.fec.gov/freedom-information-act/ redirect
 rewrite ^/press/index.shtml https://www.fec.gov/press/ redirect;
 rewrite ^/press/news_releases.shtml https://www.fec.gov/updates/?update_type=press-release redirect;
 rewrite ^/press/press_contacts.shtml https://www.fec.gov/press/#contact redirect;
+rewrite ^/press/PressOffice.xml https://www.fec.gov/updates/ redirect;
```

Comment:
Redirect to https://www.fec.gov/updates/?update_type=weekly-digest

Reply:
Going to keep as is